DigitalOffice/Tests/TimeManagementServiceTests/Validators/CreateWorkTimeRequestValidatorTests.cs

```diff
@@ -0,0 +1,188 @@
+using FluentValidation;
+using FluentValidation.TestHelper;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using LT.DigitalOffice.TimeManagementService.Database.Entities;
+using LT.DigitalOffice.TimeManagementService.Models;
+using LT.DigitalOffice.TimeManagementService.Repositories.Filters;
+using LT.DigitalOffice.TimeManagementService.Repositories.Interfaces;
+using LT.DigitalOffice.TimeManagementService.Validators;
+
+namespace LT.DigitalOffice.TimeManagementServiceUnitTests.Validators
+{
+    public class CreateWorkTimeRequestValidatorTests
+    {
+        private Mock<IWorkTimeRepository> repositoryMock;
+
+        private IValidator<CreateWorkTimeRequest> validator;
+
+        private CreateWorkTimeRequest request;
+
+        [SetUp]
+        public void Setup()
+        {
+            repositoryMock = new Mock<IWorkTimeRepository>();
+
+            repositoryMock.Setup(x => x.GetUserWorkTimes(It.IsAny<Guid>(), It.IsAny<WorkTimeFilter>()))
+                .Returns(new List<DbWorkTime>());
+
+            validator = new CreateWorkTimeRequestValidator(repositoryMock.Object);
+
+            request = new CreateWorkTimeRequest
+            {
+                WorkerUserId = Guid.NewGuid(),
+                StartTime = DateTime.Now,
+                EndTime = DateTime.Now.AddHours(6),
+                Title = "Worked...",
+                ProjectId = Guid.NewGuid(),
+                Description = "Did something"
+            };
+        }
+
+        [Test]
+        public void SuccessfulWorkTimeValidation()
+        {
+            var result = validator.Validate(request);
+            Assert.IsTrue(result.IsValid);
```

Comment:
What do you think about this writing? 
![image](https://user-images.githubusercontent.com/61093487/89463840-22e98c80-d778-11ea-91a4-73ec4013c866.png)

Reply:
It doesn't matter to me what to use, I'll change it to your version.

Reply:
Fixed.